call_tracking/migrations/0001_initial.py

```diff
@@ -1,34 +1,36 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
+# Generated by Django 3.2 on 2021-12-14 16:56
 
-from django.db import models, migrations
+from django.db import migrations, models
+import django.db.models.deletion
 import phonenumber_field.modelfields
 
 
 class Migration(migrations.Migration):
 
+    initial = True
+
     dependencies = [
     ]
 
     operations = [
         migrations.CreateModel(
             name='LeadSource',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', primary_key=True, auto_created=True, serialize=False)),
-                ('name', models.CharField(max_length=100, blank=True)),
-                ('incoming_number', phonenumber_field.modelfields.PhoneNumberField(max_length=128, unique=True)),
-                ('forwarding_number', phonenumber_field.modelfields.PhoneNumberField(max_length=128, blank=True)),
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(blank=True, help_text='E.g. "Downtown billboard"', max_length=100)),
+                ('incoming_number', phonenumber_field.modelfields.PhoneNumberField(help_text='A phone number purchased through Twilio', max_length=128, unique=True)),
+                ('forwarding_number', phonenumber_field.modelfields.PhoneNumberField(blank=True, help_text='People who call this lead source will be connected with this phone number. Must include international prefix - e.g. +1 555 555 55555', max_length=128)),
```

Comment:
Should we use a lintern/formatter to help us break these lines?

Reply:
Since it's a migration created automatically by Django do you think this should be formatted?